spec/pundit_spec.rb

```diff
@@ -25,6 +25,14 @@
       expect(Pundit.authorize(user, post, :update?)).to be_truthy
     end
 
+    it "returns the record on successful authorization" do
+      expect(Pundit.authorize(user, post, :update?)).to be(post)
+    end
```

Comment:
What is the return value in the case of a headless policy? https://github.com/varvet/pundit#headless-policies Is it worth having a test for that here?

Also in the case where there is no instance (`authorize Post`): https://github.com/varvet/pundit#policies

Ideally there would already be test cases for these and it shouldn't be your responsibility to implement them, but since we're changing the behaviour, we should probably make sure that we're not breaking any existing behaviour.

Reply:
I agree with what you are saying.